### PR TITLE
Update Couchbase Server 4 to 4.6.5

### PR DESCRIPTION
**Pull request by ceejatec**

### CODE REVIEW

**Comment by ceejatec** [comment]
Hmm... I updated our library entry to use https: instead of git:, but the new error makes no sense to me:

$ bashbrew list --build-order couchbase
error: failed sorting entries list for "couchbase"
exit status 128

Help?

File: library/couchbase [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+5.1.0: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
+enterprise-5.1.0: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.1.0
 
-5.5.0-beta: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.5.0-beta
+5.5.0-beta: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/5.5.0-beta
 
-4.6.4: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/4.6.4
-enterprise-4.6.4: git://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/4.6.4
+4.6.5: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/4.6.5
+enterprise-4.6.5: https://github.com/couchbase/docker@932a91668a12f59ee081dd0e7b3ca9235ef99403 enterprise/couchbase-server/4.6.5
```
Comment (yosifkit) [inline library/couchbase]: Most likely it is caused by this new version not existing in the old commit.

You might want to switch to the newer RFC2822 based format; it has much less repetition. You could then add multiple architectures later :wink:.

Comment (ceejatec) [reply library/couchbase]: Augh, not sure how I did that. Re-proposing. I'll look at the format change at a later date.

Comment (ceejatec) [reply library/couchbase]: Thanks for your help!

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index c4fc709..73eb794 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,5 +1,5 @@
 couchbase:5.5.0-beta @ amd64
 couchbase:community-4.5.1 @ amd64
 couchbase:community-5.0.1 @ amd64
-couchbase:enterprise-4.6.4 @ amd64
-couchbase:enterprise-5.1.0 @ amd64
+couchbase:enterprise-4.6.5 @ amd64
+couchbase:enterprise-5.1.1 @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index a36eefa..3059b2e 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,10 +1,10 @@
-couchbase:4.6.4
-couchbase:5.1.0
+couchbase:4.6.5
+couchbase:5.1.1
 couchbase:5.5.0-beta
 couchbase:community
 couchbase:community-4.5.1
 couchbase:community-5.0.1
 couchbase:enterprise
-couchbase:enterprise-4.6.4
-couchbase:enterprise-5.1.0
+couchbase:enterprise-4.6.5
+couchbase:enterprise-5.1.1
 couchbase:latest
diff --git a/couchbase_5.5.0-beta/Dockerfile b/couchbase_5.5.0-beta/Dockerfile
index fd7f4b8..dbd81b9 100644
--- a/couchbase_5.5.0-beta/Dockerfile
+++ b/couchbase_5.5.0-beta/Dockerfile
@@ -20,7 +20,7 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 ARG CB_VERSION=5.5.0-beta
-ARG CB_RELEASE_URL=http://packages.couchbase.com/releases
+ARG CB_RELEASE_URL=https://packages.couchbase.com/releases
 ARG CB_PACKAGE=couchbase-server-enterprise_5.5.0-beta-ubuntu16.04_amd64.deb
 ARG CB_SHA256=3d4eb959d51f26956ba7302a4e3fabfb922f0c8e4986c7ad52b570d13d617f26
 
diff --git a/couchbase_community-4.5.1/Dockerfile b/couchbase_community-4.5.1/Dockerfile
index 483a1a7..f829aeb 100644
--- a/couchbase_community-4.5.1/Dockerfile
+++ b/couchbase_community-4.5.1/Dockerfile
@@ -20,7 +20,7 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 ARG CB_VERSION=4.5.1
-ARG CB_RELEASE_URL=http://packages.couchbase.com/releases
+ARG CB_RELEASE_URL=https://packages.couchbase.com/releases
 ARG CB_PACKAGE=couchbase-server-community_4.5.1-ubuntu14.04_amd64.deb
 ARG CB_SHA256=de983d0137bd2de2608e52cbfdf01de6dd9d3c1d9bc45bd0702d253245a8a234
 
diff --git a/couchbase_community-5.0.1/Dockerfile b/couchbase_community-5.0.1/Dockerfile
index ac41757..83dc514 100644
--- a/couchbase_community-5.0.1/Dockerfile
+++ b/couchbase_community-5.0.1/Dockerfile
@@ -20,7 +20,7 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 ARG CB_VERSION=5.0.1
-ARG CB_RELEASE_URL=http://packages.couchbase.com/releases
+ARG CB_RELEASE_URL=https://packages.couchbase.com/releases
 ARG CB_PACKAGE=couchbase-server-community_5.0.1-ubuntu16.04_amd64.deb
 ARG CB_SHA256=44570a34323934a9e668787c26b13e8556e678de2de15052e383e5573cf34931
 
diff --git a/couchbase_enterprise-4.6.4/Dockerfile b/couchbase_enterprise-4.6.5/Dockerfile
similarity index 91%
rename from couchbase_enterprise-4.6.4/Dockerfile
rename to couchbase_enterprise-4.6.5/Dockerfile
index 526dc38..2e7d3cb 100644
--- a/couchbase_enterprise-4.6.4/Dockerfile
+++ b/couchbase_enterprise-4.6.5/Dockerfile
@@ -19,10 +19,10 @@ RUN apt-get update && \
     apt-get autoremove && apt-get clean && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-ARG CB_VERSION=4.6.4
-ARG CB_RELEASE_URL=http://packages.couchbase.com/releases
-ARG CB_PACKAGE=couchbase-server-enterprise_4.6.4-ubuntu14.04_amd64.deb
-ARG CB_SHA256=127f77825831f32cfa69704c699388413ae3b6f34dfd5eb1cb0fdb29e6a73579
+ARG CB_VERSION=4.6.5
+ARG CB_RELEASE_URL=https://packages.couchbase.com/releases
+ARG CB_PACKAGE=couchbase-server-enterprise_4.6.5-ubuntu14.04_amd64.deb
+ARG CB_SHA256=f1629c70ea9a13f88ddf70923dcfb6e05f9840a1bfee0b7150a80c930d25917f
 
 ENV PATH=$PATH:/opt/couchbase/bin:/opt/couchbase/bin/tools:/opt/couchbase/bin/install
 
diff --git a/couchbase_enterprise-4.6.4/scripts/dummy.sh b/couchbase_enterprise-4.6.5/scripts/dummy.sh
similarity index 100%
rename from couchbase_enterprise-4.6.4/scripts/dummy.sh
rename to couchbase_enterprise-4.6.5/scripts/dummy.sh
diff --git a/couchbase_enterprise-4.6.4/scripts/entrypoint.sh b/couchbase_enterprise-4.6.5/scripts/entrypoint.sh
similarity index 100%
rename from couchbase_enterprise-4.6.4/scripts/entrypoint.sh
rename to couchbase_enterprise-4.6.5/scripts/entrypoint.sh
diff --git a/couchbase_enterprise-4.6.4/scripts/run b/couchbase_enterprise-4.6.5/scripts/run
similarity index 100%
rename from couchbase_enterprise-4.6.4/scripts/run
rename to couchbase_enterprise-4.6.5/scripts/run
diff --git a/couchbase_enterprise-5.1.0/Dockerfile b/couchbase_enterprise-5.1.1/Dockerfile
similarity index 91%
rename from couchbase_enterprise-5.1.0/Dockerfile
rename to couchbase_enterprise-5.1.1/Dockerfile
index 232d44c..3591fb4 100644
--- a/couchbase_enterprise-5.1.0/Dockerfile
+++ b/couchbase_enterprise-5.1.1/Dockerfile
@@ -19,10 +19,10 @@ RUN apt-get update && \
     apt-get autoremove && apt-get clean && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-ARG CB_VERSION=5.1.0
-ARG CB_RELEASE_URL=http://packages.couchbase.com/releases
-ARG CB_PACKAGE=couchbase-server-enterprise_5.1.0-ubuntu16.04_amd64.deb
-ARG CB_SHA256=4d6a1f159577f283f6f980f6ab9161630eb2d8fd228429029de004b1be46ad76
+ARG CB_VERSION=5.1.1
+ARG CB_RELEASE_URL=https://packages.couchbase.com/releases
+ARG CB_PACKAGE=couchbase-server-enterprise_5.1.1-ubuntu16.04_amd64.deb
+ARG CB_SHA256=058acc6567db7acd8dcb80aa55a7a1de1b318848255bcb2878b285c0e52ecc8c
 
 ENV PATH=$PATH:/opt/couchbase/bin:/opt/couchbase/bin/tools:/opt/couchbase/bin/install
 
diff --git a/couchbase_enterprise-5.1.0/scripts/dummy.sh b/couchbase_enterprise-5.1.1/scripts/dummy.sh
similarity index 100%
rename from couchbase_enterprise-5.1.0/scripts/dummy.sh
rename to couchbase_enterprise-5.1.1/scripts/dummy.sh
diff --git a/couchbase_enterprise-5.1.0/scripts/entrypoint.sh b/couchbase_enterprise-5.1.1/scripts/entrypoint.sh
similarity index 100%
rename from couchbase_enterprise-5.1.0/scripts/entrypoint.sh
rename to couchbase_enterprise-5.1.1/scripts/entrypoint.sh
diff --git a/couchbase_enterprise-5.1.0/scripts/run b/couchbase_enterprise-5.1.1/scripts/run
similarity index 100%
rename from couchbase_enterprise-5.1.0/scripts/run
rename to couchbase_enterprise-5.1.1/scripts/run
```

</details>

**Comment by yosifkit** [comment]
Build test of #4420; 6dcf9e0a; `amd64` (`couchbase`):

```console

$ bashbrew build couchbase:latest
Building bashbrew/cache:147cc97a68c8e1e565a38aa70ba8355e59f10c6a1c5623d5657935fdfe498c74 (couchbase:latest)
Tagging couchbase:latest
Tagging couchbase:enterprise
Tagging couchbase:5.1.1
Tagging couchbase:enterprise-5.1.1

$ test/run.sh couchbase:latest
testing couchbase:latest
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build couchbase:5.5.0-beta
Building bashbrew/cache:7e00c77478cfabb9f27979dd30019fa2b673d7fa2416918d988cdebe85503851 (couchbase:5.5.0-beta)
Tagging couchbase:5.5.0-beta

$ test/run.sh couchbase:5.5.0-beta
testing couchbase:5.5.0-beta
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build couchbase:4.6.5
Building bashbrew/cache:85a30d7e5368e7c90f5a1040d404e9d66dfcc524ba847902b92d50ed76f3e843 (couchbase:4.6.5)
Tagging couchbase:4.6.5
Tagging couchbase:enterprise-4.6.5

$ test/run.sh couchbase:4.6.5
testing couchbase:4.6.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build couchbase:community
Building bashbrew/cache:7af8e73d12d55f0d98eb6292fd2b9518cc9e6f094c32e76db6c085d2d3cdbeb2 (couchbase:community)
Tagging couchbase:community
Tagging couchbase:community-5.0.1

$ test/run.sh couchbase:community
testing couchbase:community
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build couchbase:community-4.5.1
Building bashbrew/cache:ad48ca25ecd372d70cec214a206e96156f02822c2f89dc2207715b508962c410 (couchbase:community-4.5.1)
Tagging couchbase:community-4.5.1

$ test/run.sh couchbase:community-4.5.1
testing couchbase:community-4.5.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Comment by ceejatec** [comment]
Thanks again for your help!

**Pull request merged**